examples/opengl2_example/imgui_impl_glfw_gl2.cpp

```diff
@@ -318,7 +343,9 @@ void ImGui_ImplGlfwGL2_NewFrame()
 
     // Setup inputs
     // (we already got mouse wheel, keyboard keys & characters from glfw callbacks polled in glfwPollEvents())
+#ifndef __EMSCRIPTEN__ // Bug in emscripten library_glfw.js: GLFW_FOCUSED allways 0
     if (glfwGetWindowAttrib(g_Window, GLFW_FOCUSED))
+#endif
```

Comment:
This should be justified more thoroughly, the and code can be written with a single point of interference.

e.g.
```
#ifndef __EMSCRIPTEN__
const bool focused = glfwGetWindowAttrib(g_Window, GLFW_FOCUSED);
#else 
const bool focused = true;
#endif 
if (focused)
...
```